src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -453,32 +453,50 @@ export class JupyterNotebookBase implements INotebook {
-        // tslint:disable-next-line:no-multiline-string
-        const versionCells = await this.executeSilently(`import sys\r\nsys.version`);
-        // tslint:disable-next-line:no-multiline-string
-        const pathCells = await this.executeSilently(`import sys\r\nsys.executable`);
-        // tslint:disable-next-line:no-multiline-string
-        const notebookVersionCells = await this.executeSilently(`import notebook\r\nnotebook.version_info`);
-
-        // Both should have streamed output
-        const version = versionCells.length > 0 ? this.extractStreamOutput(versionCells[0]).trimQuotes() : '';
-        const notebookVersion =
-            notebookVersionCells.length > 0 ? this.extractStreamOutput(notebookVersionCells[0]).trimQuotes() : '';
-        const pythonPath = versionCells.length > 0 ? this.extractStreamOutput(pathCells[0]).trimQuotes() : '';
-
-        // Combine this data together to make our sys info
-        return {
-            data: {
-                cell_type: 'messages',
-                messages: [version, notebookVersion, pythonPath],
-                metadata: {},
-                source: []
-            },
-            id: uuid(),
-            file: '',
-            line: 0,
-            state: CellState.finished
-        };
+        const language = getKernelConnectionLanguage(this._executionInfo.kernelConnectionMetadata) || PYTHON_LANGUAGE;
+        if (language === PYTHON_LANGUAGE) {
+            // tslint:disable-next-line:no-multiline-string
+            const versionCells = await this.executeSilently(`import sys\r\nsys.version`);
+            // tslint:disable-next-line:no-multiline-string
+            const pathCells = await this.executeSilently(`import sys\r\nsys.executable`);
+            // tslint:disable-next-line:no-multiline-string
+            const notebookVersionCells = await this.executeSilently(`import notebook\r\nnotebook.version_info`);
+
+            // Both should have streamed output
+            const version = versionCells.length > 0 ? this.extractStreamOutput(versionCells[0]).trimQuotes() : '';
+            const notebookVersion =
+                notebookVersionCells.length > 0 ? this.extractStreamOutput(notebookVersionCells[0]).trimQuotes() : '';
+            const pythonPath = versionCells.length > 0 ? this.extractStreamOutput(pathCells[0]).trimQuotes() : '';
+
+            // Combine this data together to make our sys info
+            return {
+                data: {
+                    cell_type: 'messages',
+                    messages: [version, notebookVersion, pythonPath],
+                    metadata: {},
+                    source: []
+                },
+                id: uuid(),
+                file: '',
+                line: 0,
+                state: CellState.finished
+            };
+        } else {
+            // tslint:disable-next-line: no-suspicious-comment
+            // TODO: Provide a way for other languages to return this information
```

Comment:
**We already have all of the information we need:**
The kernelSpec info is what contains this information.
This is the first request to the Jupyter kernel by the Jlab API.

**Here's the content for Julia**
![Screen Shot 2020-11-13 at 16 36 50](https://user-images.githubusercontent.com/1948812/99133352-7a489580-25ce-11eb-937c-d5717327620a.png)

**Here's the content for Java**
```json
    {'text': 'BeakerX Home', 'url': 'http://BeakerX.com'}, 
	{'text': 'File an Issue', 'url': 'https://github.com/twosigma/beakerx/issues/new'}, 
    {'text': 'Two Sigma Open Source', 'url': 'http://opensource.twosigma.com/'}, 
    {'text': 'BeakerX JavaDoc', 'url': 'javadoc/index.html'}
], 
	'implementation': 'java', 
    'banner': 'BeakerX 1.4.0\nJava 11.0.8\nhash f8c52c4\nbuild time 2019-03-13 19:28 EDT',
	'beakerx': True, 
	'implementation_version': 'BeakerX 1.4.0\n%s %s\nhash f8c52c4\nbuild time 2019-03-13 19:28 EDT'
```

**Looks like the banner is what we need to display**
The version information is already displayed in the kernel.

Reply:
That does sound better. I'll try that and see what happens.

Reply:
Yeah not sure. It works pretty good for python. Julia is weird:

![image](https://user-images.githubusercontent.com/19672699/99134608-5c316400-25d3-11eb-9910-67b4528ccb80.png)

First banner is from julia.

Second from the python kernel.

Reply:
Maybe I should include the help links too

Reply:
Do we need the message `Start xyz kernel`?
One less line, considering the fact that you're planning on adding the help lnks as well.

Reply:
That's too wordy I think

![image](https://user-images.githubusercontent.com/19672699/99134723-d366f800-25d3-11eb-872a-045d46888894.png)

Reply:
Lol, agreed.

Reply:
I think this would be something that can be displayed on the left hand side (jupyter panel).
Nice place, pity we don't expose the kernel to extension authors, if we did, then an extension could display a WebView in the sidebar with & all the links would light up... 

😉  - exposing jupyter lab kernel api to extension authors, something we briefly touched on

Reply:
Yeah I think we can do a lot if we expose kernels/interactive window execution/ cell ranges.

Reply:
looking forward to that day, got so many ideas... want to light up this Jupyter panel & make it look like a christmas tree.. plenty of bells and whistles.. 😄